slave/base/changelog

```diff
@@ -1,3 +1,9 @@
+ -- Michal Stava <stavamichal@gmail.com>  Fri, 20 Oct 2017 09:50:00 +0200
```

Comment:
This line should be after the last part on a changelog entry.

Reply:
Fixed.